src/server/pfs/server/api_server.go

```diff
@@ -307,6 +307,7 @@ func (a *apiServer) DeleteBranch(ctx context.Context, request *pfs.DeleteBranchR
 }
 
 func (a *apiServer) ModifyFile(server pfs.API_ModifyFileServer) (retErr error) {
```

Comment:
This is going to be refactored to use the fileset API / functionality under the hood right?

Reply:
Yeah that's the plan.  I'm running into some problems with how commits are referenced, I think I screwed up branches somehow.  I don't want to refactor until that's passing.  I also just fixed something with commits in transactions which may prevent us from refactoring this too much.